docker-compose.yml

```diff
@@ -1,4 +1,46 @@
 version: '3'
+
+
 services:
-  hub:
-    image: jupyterhub/jupyterhub
\ No newline at end of file
+  backend:
+    build:
+      context: backend/
+      dockerfile: Dockerfile
+    command: sh -c "python src/manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - './backend/:/backend/'
+    ports:
+      - 8000:8000
+    depends_on:
+      - mongodb
+      
+  mongodb:
+    image: mongo:4.0.8
+    container_name: mongodb
+    restart: unless-stopped
+    command: mongod --auth
+    env_file:
+        - ./backend/.env
+    volumes:
+      - mongodbdata:/data/db
+    ports:
+    - "27017:27017"
+
+  frontend:
+    tty: true
+    build:
+      context: frontend/
+      dockerfile: Dockerfile
+    volumes:
+      - './frontend/:/frontend/'
+      - '/frontend/node_modules'
+    ports:
+      - 3000:3000
+    environment:
+      - CI=true
```

Comment:
Why is this true?

Reply:
There are some issues with `react-scripts`, and even after using `tty: true` on Linux machines, the frontend build was failing. Although, on my `wsl` it was working fine. I have downgraded the react-scripts version and set `CI=true` as one of the users suggested that using it fixes the issue[[Link](https://github.com/facebook/create-react-app/issues/8688#issuecomment-602084087)]. I assumed that it might be a solution for Linux machines. 

@mathemancer could you please confirm you are able to set up the development version or not.